seed/views/v3/geocode.py

```diff
@@ -123,7 +123,7 @@ def confidence_summary(self, request):
 
         if taxlot_view_ids:
             taxlot_views = TaxLotView.objects.filter(
-                id__in=taxlot_view_ids,
+                state_id__in=taxlot_view_ids,
```

Comment:
Let's undo these changes. We don't want to make changes to v3 to make the tests pass. Instead, let's make more changes to the tests to accommodate how the API endpoints were changed from v2 to v3.